freqtrade/commands/arguments.py

```diff
@@ -65,7 +65,7 @@
 
 ARGS_CONVERT_TRADES = ["pairs", "timeframes", "exchange", "dataformat_ohlcv", "dataformat_trades"]
 
-ARGS_LIST_DATA = ["exchange", "dataformat_ohlcv", "pairs"]
```

Comment:
Will this limit getting the mark/index price candles to what trading mode your using? I feel like they could be useful for spot, just as a reference